### PR TITLE
Auth/retrieve oauth data

### PR DESCRIPTION
**Pull request by Cavonstavant**

### CODE REVIEW

**Review by aurelenc — APPROVED** [review]

**Review by majent4 — APPROVED** [review]

File: client/src/pages/Login.tsx [hunk 1]
```diff
@@ -52,6 +56,16 @@ const Login: React.FC = () => {
         setMessage
     ] = useState<string>('');
 
+    const [
+        cookies,
+        setCookie,
+    ] = useCookies(['token']);
```
Comment (EmileEttel) [inline client/src/pages/Login.tsx:62]: Probably should put this in a utils file so that it doesn't have to be declared multiple times

**Review by EmileEttel — APPROVED** [review]
LGTM

File: client/src/pages/Login.tsx (hunk 1 above)
Comment (Cavonstavant) [reply client/src/pages/Login.tsx:62]: I did not know how to do that, please kindly pull my code on your local branch, try to fix it and i'll gladly accept the changes

Comment (EmileEttel) [reply client/src/pages/Login.tsx:62]: Nvm, my idea didn't work out

**Pull request merged**